proto/v2/decode_proto.go

```diff
@@ -16,7 +16,7 @@
 Package zipkin_proto3 adds support for the Zipkin protobuf definition to allow
 Go applications to consume model.SpanModel from protobuf serialized data.
 */
-package zipkin_proto3
+package v2
```

Comment:
While package name is not too important outside the scope of this codebase I am curious about the reason for changing the name. In any case we might need to change the description in line 16: https://github.com/openzipkin/zipkin-go/pull/165/files#diff-aed4b0e2183470649d6be446904a6842R16

Reply:
Yeah now protoc requires a `go_package` in the proto file, and the code is automatically generated into the folder with the same package name so this changed. I updated the description, but let me know if there could be any other issues.